docssrc/src/chatpreview.md

```diff
@@ -75,13 +75,13 @@ String fullInput();
 T parsedInput();
 ```
 
-`parsedInput()` is similar to `input()`, except it has been parsed by the CommandAPI's argument parser. This is a representation of what the argument in the executor would look like. For a `ChatArgument` the return type is `BaseComponent[]`, and for `AdventureChatArgument` the return type is `Component`.
+`parsedInput()` is similar to `input()`, except it has been parsed by the CommandAPI's argument parser. This is a representation of what the argument in the executor would look like. For the Spigot-based `ChatArgument` the return type is `BaseComponent[]`, and for the Adventure-based`ChatArgument` the return type is `Component`.
```

Comment:
Just a tiny typo, it looks like there's a space missing.
```suggestion
`parsedInput()` is similar to `input()`, except it has been parsed by the CommandAPI's argument parser. This is a representation of what the argument in the executor would look like. For the Spigot-based `ChatArgument` the return type is `BaseComponent[]`, and for the Adventure-based `ChatArgument` the return type is `Component`.
```
<img width="762" alt="Screen Shot 2023-09-25 at 10 14 12 AM" src="https://github.com/JorelAli/CommandAPI/assets/46540330/c303343f-d544-43c0-b322-fa6438bb6ebd">